medicines/import/src/spc_pil.rs

```diff
@@ -68,12 +75,19 @@ pub fn import<'a>(dir: &Path, client: Client, mut core: Core, verbosity: i8) ->
                     continue;
                 }
 
-                storage::upload(&hash, &client, &mut core, &file_data, &metadata, verbosity)?;
+                if false == dryrun {
```

Comment:
again, is this or `dryrun == false` more readable?

Reply:
Same as above, it's subjective. I like Yoda conditions, someone else will prefer something else.